### ISSUE TITLE
Zig

### ISSUE THREAD

**Issue by petertseng** [exercism/request-new-language-track#75]
**What programming language should we add?**

Zig

**What is the official website for the language?**

https://ziglang.org/

**Is this a language that comes in many variants? If so, which variant should we support?**

No variants known to the author of this issue.

**Does the language have an official logo?** If so
- please include a link to an example
- please list the attribution rights of that logo

https://github.com/ziglang/logo, Creative Commons Attribution-ShareAlike 4.0

**Does the language have an unofficial logo?** If so
- please explain the ways in which people use the unofficial logo
- please include a link to an example
- please list the attribution rights of that logo

No unofficial logo known to the author of this issue.

**Is there a testing framework available for the language?**

Built-in. https://ziglang.org/documentation/master/#Zig-Test

**Is this language listed as 'supported' by [Prism.js]([http://prismjs.com/#languages-list)?** If so
- what is Prism's canonical spelling for the language?
- what file extensions should be highlighted?

Zig is known to Prism as `zig`. Highlight .zig extensions.

~~**If it is not supported by Prism.js, what is the closest supported language it maps to?**~~

**Who will be leading the effort to launch the track?**

I will not be doing that, and I will be closing this issue immediately after filing it. This issue is not an actual request to start this track, but instead a report of what happened when I tried to add a Hello World exercise to a potential Zig track. This may be of use to a future maintainer who wishes to start this track. You can find the result at http://github.com/petertseng/exercism-zig - the last known status from my attempts is documented in the README there.

It looks like there are no particularly troublesome technical obstacles to creating a Zig track. A Hello World exercise has been confirmed to work, and the Travis CI setup is complete.

Since I lack the time to do so, I am not currently moving forward with efforts to pursue this track, but I present the results in case a future maintainer is able to pick up where this left off.

**Comment by massivelivefun** [comment]
Yes, as @petertseng's mentioned issue suggests I would like to spearhead, or heavily contribute to this hypothetical Zig track to some capacity. I've been making some headway with many exercises that could be used, such as resistor-color, difference-of-squares, pangram, and the like.

There are some catches, because I am completely not sure about the severity of what it means to be a maintainer for a track. So basically, I'd like to know the responsibilities that a track maintainer has to carry.

Otherwise, I'll be putting all of my effort on this fork: https://github.com/massivelivefun/exercism-zig

**Comment by Mouvedia** [comment]
https://github.com/ganderzz/zig-exercism
It's probably not up to date but you might get some help from him.

**Comment by iHiD** [comment]
Hello. Let's make a zig track then! 😄 

Responsibilities of a maintainer creating a new track are fundamentally to get an initial set of exercises in place. Also writing a test-runner will allow the track to function fully on Exercism v3 (launching early this year). But getting the exercises in place is the first step.

@massivelivefun If you're up for it, we'll create a repo for you and connect you to the wider team on our Slack. Let me know :)

**Comment by massivelivefun** [comment]
I'm up for the challenge. This whole ordeal is out of comfort zone, but I want to push myself to write good code for any reason.

**Comment by iHiD** [comment]
Excellent. Could you email erik@exercism.io please and @ErikSchierboom will get you set up :)

**Comment by kytrinyx** [comment]
I've created the repo here: https://github.com/exercism/zig/issues/1

Have fun! ❤️

**Comment by ErikSchierboom** [comment]
I've sent an email with some new track guidance to you @massivelivefun. Thanks for building this track!
